### PR TITLE
Fix unaccepted incidents showing in doc details 146606131

### PR DESCRIPTION
**Pull request by jgoley**
[PT Bug](https://www.pivotaltracker.com/story/show/146606131)
Also adds counts for document locations and incidents in document details and incident count in inbox document details.

I also updated the wording of a few helpers from source to document to I wonder if we should start renaming references to documents in the code? We refer to them as _documents_, _articles_ and _sources_. 😕

### CODE REVIEW

File: server/publications.coffee [hunk 1]
```diff
@@ -37,6 +37,7 @@ Meteor.publish 'articleIncidents', (articleId) ->
   Incidents.find
     articleId: articleId
     deleted: $in: [null, false]
+    accepted: true
```
Comment (nathanathan) [inline server/publications.coffee]: The unaccepted incidents are used on the curator inbox to underline potential counts.

Comment (jgoley) [inline server/publications.coffee]: Oh yeah, my bad. I'll update the query that feeds the table.

Comment (jgoley) [inline server/publications.coffee]: Or only filter out deleted in the publication, rather.

**Comment by nathanathan** [comment]
The incidents aren't showing at all now.

**Comment by jgoley** [comment]
@nathanathan Sorry about that, I suppose it was a merge error.

**Comment by nathanathan** [comment]
👍

**Pull request merged**